activities/models.py

```diff
@@ -121,6 +124,13 @@ class PersonAdoptPractice(CocoModel):
     verification_status = models.IntegerField(choices=ADOPTION_VERIFICATION, default=0, validators=[MaxValueValidator(2)])
     non_negotiable_check = models.CharField(max_length=256, blank=True, null=True)
     verified_by = models.IntegerField(choices=VERIFIED_BY, null=True, blank=True, validators=[MaxValueValidator(2)])
+    parentcategory = models.ForeignKey(ParentCategory, null=True, blank=True)
```

Comment:
Since we already have video in adoption table, do we need to have parentcategory in this table? Can't we back track and retrieve it?